src/main/java/org/opentripplanner/street/search/state/StateData.java

```diff
@@ -82,38 +87,57 @@ public static List<StateData> getInitialStateDatas(
       request.mode(),
       request.arriveBy(),
       request.rental().allowArrivingInRentedVehicleAtDestination(),
-      false,
       stateDataConstructor
     );
   }
 
   /**
-   * Returns an initial StateData based on the options from the RouteRequest. This returns always
-   * only a single state, which is considered the "base case"
+   * Returns an initial StateData based on the options from the {@link StreetSearchRequest}. This returns always
+   * only a single state, which is considered the "base case", should there be several possible for
+   * the given {@code request}.
    */
-  public static StateData getInitialStateData(StreetSearchRequest request) {
+  public static StateData getBaseCaseStateData(StreetSearchRequest request) {
     var stateDatas = getInitialStateDatas(
       request.mode(),
       request.arriveBy(),
       request.rental().allowArrivingInRentedVehicleAtDestination(),
-      true,
       StateData::new
     );
-    if (stateDatas.size() != 1) {
-      throw new IllegalStateException("Unable to create only a single state");
+
+    var baseStates =
```

Comment:
Is this state or state data? Seemingly we store the `stateDatas` here (datas is probably not a real word but don't know if that is an issue).

Reply:
It's datas[sic]. 

I've renamed the variable in 0c103bc86641e57adac694966354a860715dd228.